### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by warpech**
Implements suggestions made in https://github.com/StarcounterPrefabs/Launcher/commit/1297e1b007d98cda13e2be57ae61a5595bb3ec2f#commitcomment-19627458

cc @tomalec, @ubbeK 

I have decided to take the opportunity to introduce CONTRIBUTING.md file. We can add this in all of our sample apps if you agree this is a good idea.

### CODE REVIEW

**Comment by warpech** [comment]
@ubbeK please review. Does this feel informative enough now?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+The above path is hard-coded in some of the BAT files in this repository.
+
+### How to test
+
+1. Install [Node.js](https://nodejs.org/)
+2. Run `npm install` to install the test framework
+3. Start Launcher on [http://localhost:8080/](http://localhost:8080/)
+4. Run `npm test`
```
Comment (tomalec) [inline CONTRIBUTING.md]: How about `test.bat`, `launcher_selenium_test.bat`? (BTW, why do we keep both?)

It seems they are not mentioned it `npm test` https://github.com/StarcounterPrefabs/Launcher/blob/develop/package.json#L18

**Comment by ubbeK** [comment]
> @ubbeK please review. Does this feel informative enough now?

I am satisfied with the information given.

File: CONTRIBUTING.md (hunk 1 above)
Comment (warpech) [inline CONTRIBUTING.md]: I did not make any change in this text. That is a different issue: https://github.com/StarcounterPrefabs/Launcher/issues/243

Comment (tomalec) [inline CONTRIBUTING.md]: ok

**Comment by warpech** [comment]
No further comments, so merging

**Pull request merged**